.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+#### Trying to report a security issue? ####
+👌 please report security issues to security@freecodecamp.org instead of raising a Github issue.
+We look forward to working with you. If the issue is significant we'll work on resolving it as quickly as we can. We'll be happy to mention you in a published list of security researchers that found issues in our projects if you so desire.
+
+#### Describe the bug ####
+
+A clear and concise description of what the bug is.
+
+#### To Reproduce ####
+
+Steps to reproduce the behavior. We prefer a GraphQL query or link to a graphqlbin to reproduce.
+
+*be mindful that graphbin will include HTTP headers, take care when
+setting authentication tokens.*
+
+<img src="https://raw.githubusercontent.com/freeCodeCamp/open-api/staging/.github/ISSUE_TEMPLATE/share_playground.png" alt="playground"></a>
```

Comment:
@arthurbuhl I don't think these are valid repro step for classroom mode since we don't have a graphql interface, maybe you can remove the "To Reproduce" section for now? or provide a better reproduce instruction?

Reply:
Sure. I have removed the section from bug report template for now.